### DISCUSSION TITLE
Webpack issue with json over 5.77.0

### DISCUSSION THREAD

**Discussion by passariello** [webpack/webpack#17102] [Q&A]
My application work perfectly wit Wp 5.77.0. I use some require('./X.json') and I can store correctly any kind of information.
After install 5.81.0 the system continue to show issue as loader like:

///////////////////

Module parse failed: Unexpected token (2:10)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| {
> 	"version": "2023c",
| 	"zones": [...

///////////////

**If the current behavior is a bug, please provide the steps to reproduce.**

Every time I save a file HOT got wrong and show all json as disrupt. (from everywhere)

**Expected behavior:**

Continue to work fine with hot update like the 5.77.0

**Other relevant information:**
webpack version:  5.81.0
Node.js version: 18>
Operating System: Win11
Additional tools: nope

**Comment by snitin315** [comment]
@passariello Can you share a minimal repo to reproduce your issue?

**Comment by alexander-akait** [comment]
Please provide reproducible test repo
